Doc/Tutorial/chapter_appendix.tex

```diff
@@ -86,7 +86,7 @@ \section{What the heck is a handle?}
 \end{minted}
 
 With our parsers for plain text files, under Python 3 it is
```

Comment:
"under Python 3" can be dropped

Reply:
OK, I was undecided on that one - I'll remove it.